src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1796,7 +1796,7 @@ private static IEnumerable<InstrumentationKind> ParseInstrumentationKinds(string
                 return null;
             }
 
-            if (fullPath == null || !PathUtilities.IsValidFilePath(fileName))
```

Comment:
>fullPath == null [](start = 16, length = 16)

No need to check for null?

Reply:
`IsValidFilePath` does that check.

---
In reply to: [192184799](https://github.com/dotnet/roslyn/pull/27279#discussion_r192184799) [](ancestors = 192184799)